src/lsrs/web/model/common.ts

```diff
@@ -13,10 +31,50 @@ export enum HttpStatus {
 export class ResponseWrapper<T> {
 
     readonly status: HttpStatus;
-    readonly content: T;
+    readonly content: Optional<T>;
```

Comment:
Why not a nullable property?
`readonly content?: T`

Reply:
Because that did not work with "null" (null vs undefined).

Reply:
Why would you use null?

Reply:
You won't be happy about my answer. :D